katsdptelstate/test/test_endpoint.py

```diff
@@ -79,25 +78,28 @@ def test_parser(self) -> None:
             Endpoint('10.1.0.1', 60),
             Endpoint('10.1.0.2', 60)
         ]
-        assert_equal(expected, endpoints)
+        assert expected == endpoints
 
     def test_parser_bad_count(self) -> None:
-        assert_raises(ValueError, endpoint_list_parser(1234), '192.168.0.1+-4')
+        with pytest.raises(ValueError):
+            endpoint_list_parser(1234)('192.168.0.1+-4')
 
     def test_parser_non_integer_count(self) -> None:
-        assert_raises(ValueError, endpoint_list_parser(1234), '192.168.0.1+hello')
+        with pytest.raises(ValueError):
+            endpoint_list_parser(1234)('192.168.0.1+hello')
 
     def test_parser_count_without_ipv4(self) -> None:
-        assert_raises(ValueError, endpoint_list_parser(1234), 'hello.world+4')
+        with pytest.raises(ValueError):
+            endpoint_list_parser(1234)('hello.world+4')
 
     def test_parser_single_port(self) -> None:
         parser = endpoint_list_parser(1234, single_port=True)
         endpoints = parser('hello:1234,world')
         expected = [Endpoint('hello', 1234), Endpoint('world', 1234)]
-        assert_equal(expected, endpoints)
+        assert expected == endpoints
 
     def test_parser_single_port_bad(self) -> None:
-        assert_raises(ValueError, endpoint_list_parser(1234, single_port=True), 'x:123,y:456')
+        pytest.raises(ValueError, endpoint_list_parser(1234, single_port=True), 'x:123,y:456')
```

Comment:
Oh yes, and do you have a rationale for picking the function vs the context manager version of `pytest.raises`, or am I overthinking it? 🙂

Reply:
This is just what pytestify spat out - I didn't realise pytest even supported this version. In another place I rewrote this to the context manager version, so for consistency I'll do it here too.